package/kernel/mac80211/files/lib/wifi/mac80211.sh

```diff
@@ -106,6 +108,10 @@ detect_mac80211() {
 			dev_id="set wireless.radio${devidx}.macaddr=$(cat /sys/class/ieee80211/${dev}/macaddress)"
 		fi
 
+		local labelmac=$(get_mac_label)
+		local ssid=OpenWrt_11${mode_band}
```

Comment:
Don't add bands to SSIDs, you're effectively creating two different networks, forbidding band steering etc.